modules/strap/strap.service.js

```diff
@@ -62,13 +63,7 @@ class StrapService {
   }
 
   async getStrapsByModel(id) {
-    const strap = Strap.find({ model: id }).exec();
-
-    if (!strap) {
-      throw new RuleError(STRAP_NOT_FOUND, NOT_FOUND);
-    }
-
-    return strap;
+    return Strap.find({ model: id }).exec();
```

Comment:
Delete unused code. Because "find" always return array.

Reply:
Why not consider changing it to !strap.length to check if an array is empty? Because anyway if we return an empty array I don't think this would be okay for the functionality our backend modules offer.